src/Vlingo.Http/RequestParser.cs

```diff
@@ -221,7 +223,7 @@ private string Compact()
             private Optional<string> NextLine(string errorResult, string errorMessage)
             {
                 var possibleCarriageReturnIndex = -1;
-                var lineBreak = _requestText.IndexOf("\n", _position, StringComparison.InvariantCultureIgnoreCase);
```

Comment:
The invariant culture has to be there. It should not depend on the specific culture.